operators/ip_match_from_file.go

```diff
@@ -17,14 +18,30 @@ type ipMatchFromFile struct {
 func (o *ipMatchFromFile) Init(options coraza.RuleOperatorOptions) error {
 	data := options.Arguments
 
+	dataParsed := ""
```

Comment:
This function is not on the hot path so it's not such a big deal but I think @M4tteoP also wants to learn Go idioms :)

It would be better to use `strings.Builder` since code is similar but more efficient.